src/api/interceptors/requestPascalToCamelCase/requestPascalToCamelCase.ts

```diff
@@ -0,0 +1,23 @@
+export const requestPascalToCamelCase = <TObject>(object: Record<string, any>): TObject => {
+  const changeStringToCamelCase = (key: string) =>
+    key
+      .replace(/[^a-z0-9]{1,}./gi, (match) => match.toUpperCase().substring(match.length - 1))
+      .replace(/^.{1}/, (match) => match.toLowerCase())
+      .replace(/[^a-z0-9]$/, '');
+
+  const keysTuCamelCase = (input: any): any => {
```

Comment:
Czy to `any` nie świeci przez eslinta? Generalnie powinno. (dobrze by dodać taką regułę)
Ogólna zasada - **nie używamy `any`**, nigdy. Jeśli już naprawdę się nic nie da zrobić, to używamy `unknown` (sprawdź sobie jaka jest różnica pomiędzy nimi)